client/landing/stepper/declarative-flow/site-migration-flow.ts

```diff
@@ -26,6 +27,36 @@ import type { AssertConditionResult, Flow, ProvidedDependencies } from './intern
 
 const FLOW_NAME = 'site-migration';
 
+/**
+ * Redirects users coming from landing pages and "/sites" to the new migration flow.
+ */
+const useMigrationFlowRevampRedirect = () => {
+	const [ query ] = useSearchParams();
+	const fullQueryString = query.toString();
+	const ref = query.get( 'ref' ) ?? '';
```

Comment:
NIT: Usually, it is better to use null than an empty string. 
It makes typescript tell us when we need to protect the code, and the if will be more natural, like `if(ref).`
```suggestion
	const ref = query.get( 'ref' );
```

Reply:
Good catch! I had used it because I started using `includes` to check different refs, but I changed the approach later and it's really not necessary anymore.